client/src/assets/messages/landing.ts

```diff
@@ -0,0 +1,30 @@
+// Strings for the landing page.
+
+/**
+ * Strings related to the scorecard-related part of the landing page.
+ */
+export class ScorecardSearch {
+  static HEADER = `Get your drinking water score`;
+  static BODY = `Understand the likelihood of your home or community's exposure to lead service lines.`;
+  static CTA_PLACEHOLDER = `Enter your address`;
+  static CTA_BUTTON = 'Get Score';
```

Comment:
This is super nit but either use `` or '' for consistency. Here and below

Reply:
Done.